### DISCUSSION TITLE
Question: Weekly RFC call no longer a thing?

### DISCUSSION THREAD

**Discussion by aFuzzyBear** [withastro/roadmap#9] [Proposal]
This new format for the RFC's is a great manoeuvre towards a stronger governance model around the RFC's which was sorely needed as the existing model was starting to show signs of straining under the weight. 
It should also clear up the 'Astro.Issues' section of the main repo, letting that be used to only raise support issues and be a bucket for everything. 

I would like to ask would moving the discussions for each RFC over to Github, would this herald the end of the weekly RFC calls to discuss any of the RFC's that are making their way through the proposal stages?

I understand that the developer time from the members of the core team is highly valuable and perhaps the format of the weekly rfc calls might of been also showing symptoms of the existing process; Where RFC's were being discussed but few were making it through. Holding weekly calls might of been difficult to achieve given the teams workloads, even down to the discussions themselves, there can be a more formal manner of discussion implemented, instead of the fly-by-interruptions that seemed to be happening with more frequency, (my own apologies for doing this in the past)

It does make me wonder if the moving the conversations over to 'Github' would feel like a slight dislocation away from the discord community, as it  now involves two different forms of participation. How to encourage support and promote the discussion in both places would be interesting to see, it might make the role of DevRel be even more important to the project

**Comment by FredKSchott** [comment]
Thanks for the feedback @aFuzzyBear! No change to our weekly RFC call, we just skipped last week's for the holiday.  Looking forward to seeing everyone online tomorrow!

The larger move to an RFC repo was actually mainly meant to make the RFC call more productive, and the more detailed template is meant to address some of the issues that we'd get hung up on in the past (ex: guessing at an implementation vs. having one strictly laid out).

**Reply by aFuzzyBear** [reply]
Excellent, I look forward to seeing how this progress, see you tomorrow  😀
